src/box/alter.cc

```diff
@@ -5799,7 +5239,6 @@ TRIGGER(on_replace_sequence, on_replace_dd_sequence);
 TRIGGER(on_replace_sequence_data, on_replace_dd_sequence_data);
 TRIGGER(on_replace_space_sequence, on_replace_dd_space_sequence);
 TRIGGER(on_replace_trigger, on_replace_dd_trigger);
-TRIGGER(on_replace_fk_constraint, on_replace_dd_fk_constraint);
```

Comment:
What happens when I update Tarantool from 2.10 to 2.11 *without* running `box.schema.upgrade()`. SQL constraints will stop working until I run `box.schema.upgrade()`?

Reply:
I asked @unera and we came to the conclusion that Tarantool 2.11 should not work properly with a snapshot from Tarantool 2.10 without a proper upgrade. I believe that partly to avoid such problems, we are working on the DOWNGRADE feature.

Reply:
I think it is OK

But I've just touched @Mons too, Maybe he has another other opinion

Reply:
I don't think that's about downgrade feature.

In general, such a change is not ok, because online upgrade is one of the main scenarios of upgrade.
And loss of some functionality during an upgrade could be a problem.
But currently we have only a little usecases for SQL and we hope that almost nobody uses current sql constraints.
So, for this current case let's consider, that temporary loss of constraints is ok (due to lack of dev resources to support complete online upgrade and for the sake of code simplicity)